src/app/intro-stepper/intro-stepper.component.html

```diff
@@ -27,33 +27,35 @@
       </button>
       <div class="intro-step" *ngIf="introSteps[currentStepIndex] === IntroStep.INTRO">
         <div class="image-container">
-          <img src="/alert_sign_stake.svg" alt="A woman stakes a circular traffic sign with an alert symbol on the sign. There
-                are geometric shapes with varying textures on the ground.">
+          <img src="/welcome_illustration.svg" alt="A woman waving." class="image-popup">
         </div>
         <div class="message">
-          <h2>Introducing SafeNet</h2>
-          Use this tool to document harmful content
+          <h2>Welcom to SafeNet</h2>
+          Use the tol to flag online harrasment and collect evidence about the abuse
```

Comment:
1. Check spelling.
2. Have we received new content? Why are we changing it?

Reply:
This content was not provided before. It is from the design